src/components/InventoryGroupDetail/GroupDetailInfo.js

```diff
@@ -1,19 +1,32 @@
-import { EmptyState, EmptyStateBody, Spinner } from '@patternfly/react-core';
-import { InvalidObject } from '@redhat-cloud-services/frontend-components';
+import {
+    Button,
+    Card,
+    CardTitle,
+    CardBody,
+    CardHeader,
+    CardActions } from '@patternfly/react-core';
 import React from 'react';
-import { useSelector } from 'react-redux';
 
 const GroupDetailInfo = () => {
-    const { uninitialized, loading } = useSelector((state) => state.groupDetail);
-
-    // TODO: implement according to mocks
-
     return (
-        <EmptyState>
-            <EmptyStateBody>
-                {uninitialized || loading ? <Spinner /> : <InvalidObject />}
-            </EmptyStateBody>
-        </EmptyState>
+        <Card>
+            <CardHeader>
+                <CardActions>
+                    <Button variant="secondary" isSmall="true" >
```

Comment:
Not sure why you use isSmall here. On sketch, it looks like the normal size

Reply:
It's the same size as on the sketch
![image](https://user-images.githubusercontent.com/62722417/225072587-d2b985a4-847d-42ce-8731-d68d0b377c10.png)
![image](https://user-images.githubusercontent.com/62722417/225072614-7724f325-f6ac-4b71-8fa6-cca6b1001210.png)

Reply:
Removed isSmall props